spree/core/spec/lib/spree/events_spec.rb

```
@@ -98,6 +98,54 @@
+      subscriber = ActiveSupport::Notifications.subscribe('dispatch.spree_events') do |*, payload|
+        events << payload
+      end
+      yield
+      events
+    ensure
+      ActiveSupport::Notifications.unsubscribe(subscriber)
+    end
+
+    it 'instruments every subscriber dispatch as dispatch.spree_events' do
+      handler_class = Class.new do
+        def self.name = 'Spree::Testing::DispatchHandler'
```

RuboCop (1.88.2)

[fatal] unexpected token tEQL
(Using Ruby 2.7 parser; configure using `TargetRubyVersion` parameter, under `AllCops`)

(Lint/Syntax)